### PR TITLE
add .devcontainer files to allow VSCode to use Docker for the Dev Env

### PR DESCRIPTION
**Pull request by jhamm**
add .devcontainer files to allow VSCode to use Docker for the develoment environment

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    && ./emsdk activate 2.0.6
+
+ENV PATH="/opt/dev/emsdk:/opt/dev/emsdk/node/12.9.1_64bit/bin:/opt/dev/emsdk/upstream/emscripten:${PATH}"
+
+# [Optional] Uncomment if you want to install an additional version of node using nvm
+# ARG EXTRA_NODE_VERSION=10
+# RUN su node -c "source /usr/local/share/nvm/nvm.sh && nvm install ${EXTRA_NODE_VERSION}"
+
+# [Optional] Uncomment if you want to install more global node packages
+# RUN su node -c "npm install -g <your-package-list -here>"
```
[the remark this answers is not in the document]
Comment (jhamm) [reply .devcontainer/Dockerfile:26]: No, they were added by the template. I can remove them 👍

[the remark this answers is not in the document]
Comment (xcv58) [reply .devcontainer/Dockerfile:26]: Thanks and please remove them 🙏

[the remark this answers is not in the document]
Comment (jhamm) [reply .devcontainer/Dockerfile]: Good suggestion! They are gone.

**Review by xcv58 — APPROVED** [review]

**Review by Siddhant-K-code — APPROVED** [review]

**Comment by Siddhant-K-code** [comment]
@conwnet this looks good to me, please review it once.

**Comment by conwnet** [comment]
thanks! nice work!

**Pull request merged**